docs/chia-blockchain/protocol/harvester-protocol.md

```diff
@@ -131,4 +131,27 @@ class Plot(Streamable):
     plot_public_key: G1Element
     file_size: uint64
     time_modified: uint64
+
+## Plot Filter Computation (Chia 2.5.5+)
+
+Starting in Chia 2.5.5, the harvester protocol has been optimized to ensure that plot filter computation is performed directly on the harvester for improved efficiency and reduced network overhead.
+
+### Enhanced Plot Filter Processing
+
+**Available in**: Chia 2.5.5 and later versions
+
+The harvester now performs optimized plot filter computation using the formula:
+
+`plot filter bits = sha256(plot_id + challenge_hash + sp_hash)`
+
+**Benefits of Harvester-Side Computation**:
```

Comment:
Wasn't this always done on the harvester?
https://docs.chia.net/chia-blockchain/consensus/chains/harvester-algorithm/

Reply:
yes, it seems I have accidentally crossed two different updates I was working on. I removed this portion